metadata-ingestion/src/datahub/ingestion/api/auto_work_units/auto_dataset_properties_aspect.py

```diff
@@ -0,0 +1,98 @@
+import dataclasses
+from typing import Dict, Iterable, Optional
+
+from datahub.ingestion.api.workunit import MetadataWorkUnit
+from datahub.metadata.schema_classes import (
+    DatasetPropertiesClass,
+    OperationClass,
+    TimeStampClass,
+)
+from datahub.specific.dataset import DatasetPatchBuilder
+from datahub.utilities.urns.urn import guess_entity_type
+
+
+@dataclasses.dataclass
+class TimestampPair:
+    last_modified: Optional[TimeStampClass]  # last_modified of datasetProperties aspect
```

Comment:
comments are a crutch to use if the code isn't clear. But if we can make the variable names clear and not need comments, that's better

please update the variable names to be more clear